hscontrol/types/config.go

```diff
@@ -461,8 +472,12 @@ func GetDatabaseConfig() DatabaseConfig {
 	}
 
 	return DatabaseConfig{
-		Type:  type_,
-		Debug: debug,
+		Type:                  type_,
+		Debug:                 debug,
+		SkipErrRecordNotFound: skipErrRecordNotFound,
+		SlowThreshold:         slowThreshold,
+		ParameterizedQueries:  parameterizedQueries,
+		PrepareStmt:           prepareStmt,
```

Comment:
This should be in a nested object, GormConfig.

Reply:
Got it, I agree. I'll move these to GormConfig.